engine/src/test/java/org/hibernate/search/testsupport/junit/SearchFactoryHolder.java

```diff
@@ -152,4 +154,14 @@ public BatchBackend getBatchBackend() {
 		return batchBackend;
 	}
 
+	/**
+	 * The default for the tests in hibernate-search-engine is to not rely on JPA
+	 * specific annotations. These are typically used though when integrating
+	 * with Hibernate, so some tests might want to explicitly enable this.
+	 */
+	public SearchFactoryHolder enableJPAAnnotationsProcessing(boolean enableJPAAnnotationsProcessing) {
```

Comment:
I find the new name more suited for this specific case, as it has an imperative effect on a "toggle".

You're right it's inconsistent though. I think the only (weak) justification I can think of is that the multi-tenancy and provided-id-strategies need to be plugged in additional strategies to work.